hubmap/frontend/cypress/integration/hubmap_spec.js

```diff
@@ -32,4 +32,32 @@ describe('HuBMAP', () => {
     // Footer:
     cy.contains('Supported by the NIH Common Fund');
   });
+
+  describe('The Login Page', function () {
```

Comment:
I would lean towards just adding the 
```
cy.route(`${api}/auth/login/globus`, 'fixture:login.json');
```
to the list above... but on the other hand, if we want to be sure the code is only accessing the API end-points we mean for it to access, this is better. If your intention is to have a tighter, but more verbose test, then this is good.

Reply:
So hubmap_spec appears to be written by you, however, please feel free to change this. I have just added this during my tests. We can leave this file out when merging.